### PR TITLE
updated files for user story 3854

### PR DESCRIPTION
**Pull request by bwelch8105**

### CODE REVIEW

File: cypress/Shared/MeasuresPage.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 
 export class MeasuresPage {
-
+    public static readonly measuresButton = '.styles__ExtraButton-sc-147glsa-9 dTZxVT'
```
Comment (cvasile001) [inline cypress/Shared/MeasuresPage.ts:4]: This is already located in class LandingPage

File: cypress/Shared/TestCasesPage.ts [hunk 2]
```diff
@@ -3,30 +3,51 @@ import {EditMeasurePage} from "./EditMeasurePage"
 export class TestCasesPage {
 
     public static readonly newTestCaseButton = '.sc-iqseJM'
+    public static readonly editTestCaseButton ='[class="TestCaseList__Button-sc-1iefzo5-2 cIVwpR"]'
+    ///<button data-testid="edit-test-case-61fda86b0ed1674fd78ac96b" class="TestCaseList__Button-sc-1iefzo5-2 cIVwpR">Edit</button>
+    public static readonly testCaseTitle = '[data-testid=create-test-case-title]'
     public static readonly testCaseDescriptionTextBox = '[data-testid=create-test-case-description]'
-    public static readonly createTestCaseButton = '[data-testid=create-test-case-button]'
```
Comment (cvasile001) [inline cypress/Shared/TestCasesPage.ts:7]: not sure why we need to update this element name?

File: package.json [hunk 3]
```diff
@@ -7,17 +7,25 @@
...
 		"cypress:open:test": "cypress open --env configFile=test",
 		"cypress:run": "cypress run --spec cypress/integration/**/*",
+		"windows:cypress:run": "cypress run --spec .\\cypress\\integration\\**\\*",
 		"compile": "tsc --noEmit",
 		"combine:reports": "npx mochawesome-merge 'cypress/results/*.json' > mochawesome.json",
+		"windows:combine:reports": "npx mochawesome-merge '.\\cypress\\results\\*.json' > mochawesome.json",
 		"generateOne:report": "npx marge mochawesome.json",
 		"delete:mochawesome": "rm mochawesome.json",
-		"delete:reports": "rm cypress/results/*.json",
+		"delete:reports": "del .\\cypress\\results\\*.json",
```
Comment (cvasile001) [inline package.json:16]: I would call this windows:delete:reports

File: package.json [hunk 4]
```diff
@@ -7,17 +7,25 @@
 		"cypress:open:dev": "cypress open --env configFile=dev",
 		"cypress:open:test": "cypress open --env configFile=test",
 		"cypress:run": "cypress run --spec cypress/integration/**/*",
+		"windows:cypress:run": "cypress run --spec .\\cypress\\integration\\**\\*",
 		"compile": "tsc --noEmit",
 		"combine:reports": "npx mochawesome-merge 'cypress/results/*.json' > mochawesome.json",
+		"windows:combine:reports": "npx mochawesome-merge '.\\cypress\\results\\*.json' > mochawesome.json",
 		"generateOne:report": "npx marge mochawesome.json",
 		"delete:mochawesome": "rm mochawesome.json",
```
Comment (cvasile001) [inline package.json:15]: you will also need a windows:delete:mochawesome

File: package.json [hunk 5]
```diff
@@ -7,17 +7,25 @@
...
+		"windows:dev:all:ui:tests": "cypress run --env configFile=dev -s '.\\cypress\\integration\\WebInterface\\**\\*.spec.ts' --browser chrome --headed",
 		"dev:all:services:tests": "NO_COLOR=1 $(npm bin)/cypress run --env configFile=dev --spec 'cypress/integration/Services/**/*.spec.ts' --browser chrome --headed;",
+		"windows:dev:all:services:tests": "NO_COLOR=1 $(npm bin)/cypress run --env configFile=dev --spec '.\\cypress\\integration\\Services\\**\\*.spec.ts' --browser chrome --headed;",
 		"dev:all:measure:service:tests": "NO_COLOR=1 $(npm bin)/cypress run --env configFile=dev --spec 'cypress/integration/Services/Measure Service/**/*.spec.ts' --browser chrome --headed;",
+		"windows:dev:all:measure:service:tests": "NO_COLOR=1 $(npm bin)/cypress run --env configFile=dev --spec '.\\cypress\\integration\\Services\\Measure Service\\**\\*.spec.ts' --browser chrome --headed;",
 		"dev:all:ui:tests:report": "npm run delete:reports; npm run delete:mochawesome; npm run dev:all:ui:tests; npm run combine:reports; npm run generateOne:report",
```
Comment (cvasile001) [inline package.json:29]: might be good to put all the windows scripts together

File: cypress/Shared/TestCasesPage.ts [hunk 6]
```diff
@@ -3,30 +3,51 @@ import {EditMeasurePage} from "./EditMeasurePage"
...
         cy.get(this.successMsg).should('contain.text', 'Test case saved successfully!')
 
+        //
         //Verify created test case exists on Test Case Page
         cy.get(EditMeasurePage.testCasesTab).click()
         cy.get(this.listOfTestCases).contains(testCaseDescription)
-
         cy.log('Test Case created successfully')
+
+        //Edit / Update test case title
```
Comment (cvasile001) [inline cypress/Shared/TestCasesPage.ts:35]: are you editing a test case or creating a new test case in this function?

**Review by cvasile001 — CHANGES_REQUESTED** [review]
Left some comments, but probably best is just getting together and going over it together

**Comment by bwelch8105** [comment]
creating a new pull request after cloning down latest changes and manually comparing my changes to latest

**Pull request closed without merging**